src/app/services/mocks/count.service.mock.ts

```diff
@@ -63,7 +63,13 @@ export class CountServiceMock {
     });
   }
 
-  updateAllCounts(constraint: Constraint) {
+  updateAllCounts(constraint: Constraint): Promise<any>  {
+    return new Promise<any>(resolve => {
```

Comment:
Isn't this the same as `Promise.resolve(true)`?

Reply:
Yes, looks like both work fine, we just use `return new Promise<any>(resolve => {` everywhere in the tests. Is it useful to change it in all the places?

Reply:
It's shorter, but indeed otherwise the same.